kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -105,6 +105,13 @@ a, input[type="checkbox"] {
   }
 }
 
+@-moz-document url-prefix() {
```

Comment:
Please, move it to the form section and leave a comment about special behavior in firefox

Reply:
Done